tests/core/conftest.py

```diff
@@ -283,8 +283,8 @@ def wait_until_available(client, obj, timeout=DEFAULT_TIMEOUT):
         except ApiError as e:
             if e.error.status != 403:
                 raise e
-        else:
-            return obj
+            else:
+                return obj
```

Comment:
You don't reference this function in your test and I don't think this change is necessary or correct, please revert.